src/api/registration/registration.service.js

```diff
@@ -68,7 +72,7 @@ const saveRegistration = async (registration, fsa_rn, council) => {
     }
 
     const metadata = await createMetadata(registration.metadata, reg.id);
-
+    updateCompletedInCache(fsa_rn, "registration", "success");
```

Comment:
Is there any reason why the completed property can't be a simple true/false flag?

Reply:
it was only because the tascomi could have been n/a, but if removing that have changed to true/false